src/server.h

```diff
@@ -12,11 +13,12 @@ class Server
     private:
         void start_receive();
         void handle_receive(const boost::system::error_code& error, std::size_t);
```

Comment:
Except pure type specification, you should always use parameter names. Not just types.